src/OrchardCore.Cms.Web/appsettings.json

```diff
@@ -8,7 +8,8 @@
     }
   },
   "OrchardCore": {
-
+    "DefaultCulture": "",
+    "SupportedCultures":  [ "en" ]
```

Comment:
"OrchardCore": {
      "OrchardCore.Setup": {
        "DefaultCulture": "",
        "SupportedCultures": [ "en" ]
      }

Note: If `DefaultCulture` is an empty string, at runtime you will get a null value.

Reply:
No problem because we check for null or empty in the localization middleware

Reply:
Yes for `DefaultCulture` but not done for `SupportedCultures`. I saw it failing when running unit tests and it was working with `if (supportedCultures?.Length > 0)`.

Then, to resolve your config values you need an `OrcharCore` section but also an `OrchardCore.Setup` sub section in your `appsettings.json`.

Reply:
> Then, to resolve your config values you need an OrcharCore section but also an OrchardCore.Setup sub section in your appsettings.json.

It's already there in the Web project

Reply:
As i can see in appsettings.json there is only the `OrchardCore` section.

Yes, in any case we need the OC section, then we use a module sub section if we want config values that are specific to a module.

Then, by using `IShellConfiguration` at runtime you don't need to specify the OC section but still the module sub section if there is one.